WalletWasabi/Services/UpdateManager.cs

```diff
@@ -140,11 +184,62 @@ private async Task<(Version Version, string DownloadUrl, string FileName)> GetLa
 			assetDownloadUrls.Add(asset["browser_download_url"]?.ToString() ?? throw new InvalidDataException("Missing download url from response."));
 		}
 
+		bool isReleaseValid = await ValidateWasabiSignatureAsync(softwareVersion, assetDownloadUrls).ConfigureAwait(false);
+		if (!isReleaseValid)
+		{
+			throw new InvalidOperationException($"Downloading new release was aborted, Wasabi signature was invalid.");
```

Comment:
```suggestion
			throw new InvalidOperationException($"Installing new release was aborted, Wasabi signature was invalid.");
```
The download was complete (otherwise the verification would not be possible), the install was aborted.

Do we delete the file that was not valid?

Reply:
Yes, there is a Cleanup function in case of this excpetion.

Reply:
"Installing" isn't quite true either, as we are not installing here, so it's not aborted truly.
Is "Getting/Obtaining new release" more acceptable?

Reply:
I'd say `installing` is better.